packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -55,19 +56,22 @@ export default ( blockData, baseTree, userTree ) => {
 	 * @return {Array} An array of style declarations.
 	 */
 	const getBlockStylesDeclarations = ( blockSupports, blockStyles ) => {
-		const declarations = [];
-		Object.keys( STYLE_PROPS ).forEach( ( key ) => {
-			if (
-				blockSupports.includes( key ) &&
-				get( blockStyles, STYLE_PROPS[ key ], false )
-			) {
-				declarations.push(
-					`${ key }: ${ get( blockStyles, STYLE_PROPS[ key ] ) }`
-				);
-			}
-		} );
-
-		return declarations;
+		return reduce(
+			__EXPERIMENTAL_STYLE_MAPPINGS,
+			function ( declarations, stylePath, rawKey ) {
+				const key = kebabCase( rawKey );
```

Comment:
According to [the docs](https://lodash.com/docs/#kebabCase), `kebabCase("--wp--style--color--link")` becomes `wp-style-color-link`, which causes issues both with `blockSupports` check and generating the declaration.

Reply:
Nice catch the issue was fixed 👍